ocs_ci/framework/conf/ocp_version/ocp-4.10-config.yaml

```diff
@@ -9,4 +9,6 @@ DEPLOYMENT:
   # https://docs.openshift.com/container-platform/4.7/post_installation_configuration/machine-configuration-tasks.html#machine-config-overview-post-install-machine-configuration-tasks
   ignition_version: "3.2.0"
 ENV_DATA:
+  rhel8_worker_ami: 'ami-074bab065c112399f' # RHEL 8.4
```

Comment:
Can't these be set in the default config or be set as constants instead of explicitly in each individual config?

Reply:
I moved it out of default config to be bounded to specific ocp version. E.g in 4.5 documentation there is mentioned rhel version 7.6 or something like this, from 4.6 till 4.9 ocp doc mentioning explicitly 7.9 and I guess it can change in futures versions as well, hence I would like to be explicit in each ocp config file without need to think in the future that if I will change the default image that it can break older versions deployment.

Reply:
Well for each version of RHEL we should only have one AMI that we use, correct? If that's the case then the configs can specify the _version_ of RHEL that they require and we can handle mapping that version to the actual AMI ID elsewhere so if that AMI ID ever changes we don't need to change it across a dozen config files. Instead it's a one line change. WDYT?

Reply:
We still need to have ability to change to specific version we want if we need to test specific version. I came up with the change which will be more fit to what you would like to see. 

Anyway it will require the entry to the ocp version config files once we will diverge from what is the default and we will need to e.g. to ocp 4.6 config files put: `rhel_version: "7.9"` it it will be the lates supported version.

Anyway I hope that if OCP supported on RHEL 7, every new update to minor version should probably still support also those older versions of OCP even they do not update the documentation for older version to keep version updated. Let's see how it will go.

@clacroix12 let me know if you rather like this approach. Before I went with explicit definition because we go with such approach also for the other values.

Reply:
Yeah I think this works. We can absolutely specify `rhel_version` in individual config files if the supported version differs from the default. Having the AMI ID specified in one place and being mapped to from the version is cleaner and calls for fewer changes if we need to update the AMI for any reason.